react/components/ShippingSimulator/index.js

```diff
@@ -77,7 +83,7 @@ const ShippingSimulator = ({
 
   return (
     <Fragment>
-      <div className={`${styles.shippingContainer} t-small c-on-base`}>
+      <div className={`${styles.shippingContainer} t-small c-on-base`} onKeyPress={keyPress}>
```

Comment:
A clickable div should be tabbable and semantically a button:

```suggestion
      <div className={`${styles.shippingContainer} t-small c-on-base`} onKeyPress={keyPress} tabindex="0" role="button">
```